glumpy/gloo/framebuffer.py

```diff
@@ -365,7 +365,7 @@ def _delete(self):
         """ Delete buffer from GPU """
 
         log.debug("GPU: Delete framebuffer")
-        gl.glDeleteFramebuffer(self._handle)
+        gl.glDeleteFramebuffers(1, np.array([self._handle]))
```

Comment:
Is that mandatory to transform it in a numpy array?

Reply:
I the function expects an array, I tried using a python list but got an error about some attribute missing from the `framebuffers` parameter. The [PyOpenGL docs](http://pyopengl.sourceforge.net/documentation/manual-3.0/glDeleteFramebuffers.html) are slightly unspecific in this regard, as there it says the function expects a pointer to an array.